<launch>
  <param name="device_id" value="0"/>
  <param name="calibration_path" value="$(find camera-controller)/config/camera.xml" />
  <param name="file_path" value="$(find camera-controller_PoC2020iot)/config/source_position/"/>
  <node name="camera" pkg="camera-controller" type="camera.py" output="screen"/>
  <node name="ar" pkg="camera-controller" type="ar_detector.py" output="screen"/>
  <node name="capture_position" pkg="camera-controller_PoC2020iot" type="capture_position.py" output="screen"/>
  <node name="image_view" pkg="image_view" type="image_view" output="screen">
    <remap from="image" to="image_raw" />
  </node>
</launch>
